include/README

```diff
@@ -0,0 +1,76 @@
+ODP specification
+=================
+
+ODP specification consists of several types of files, which together provide
+full list of types, values and functions that ODP implemention MUST provide.
+
+ODP API specification
+---------------------
+
+These are the files from `include/odp/api/spec` directory. They specify a set
+of functions that MUST be provided by ODP implementation. Doxygen comments
+inside these files document requirements for ABI interface provided by an
+implementation. An implementation MUST use these headers AS IS, without any
+modifications to be compatible with ODP specification.
+
```

Comment:
... a set of function prototypes, types, type names, enumerations, etc that MUST be provided ...

... document requirements for API ... (== typo: not ABI)

Reply:
Git commit log text missing for this patch

Reply:
@psavol 
Ack for the first change. For the second change I meant that spec files document requirements for ABI without providing exact declaration, Maybe you can suggest better wording?

Reply:
OK then something like this: "Content of some types and value of some enumerations are left undefined in API spec. These are defined either in ABI spec or implementation specific (non-ABI compatible) headers ."